src/App.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { useState } from "react";
```

Comment:
They can be imported on one line.
As you are using the latest version of react there is no need to explicitly import react anymore, it implicitly imported under the hood